package.json

```diff
@@ -29,7 +29,7 @@
     "preview": true,
     "activationEvents": [
         "onCommand:containerApps.viewProperties",
-        "onCommand:containerApps.openLogsInPortal",
+        "onCommand:containerApps.openInPortal",
```

Comment:
How come you changed the command name to `openInPortal`?

Reply:
Because the command should be generic enough to handle both our logs and container app nodes.  

It's kind of a long story - originally it was `openInPortal`, but when I did the unified merge, I didn't completely understand the difference between the `openInPortal` command issued by Resource Groups vs by the individual extension.  In migrating, I missed that we lost the `openInPortal` for our Container App nodes.  I thought at the time that we only needed to keep one command for our `GenericTreeItem` log nodes so I made it a separate command.  Now that I looked back, I realized that I hadn't handled it properly since the `ContainerApps` nodes do not show `openInPortal` so I'm fixing all of the open logic with one generic `openInPortal` (probably how it should have been handled the first time?)